Makefile

```diff
@@ -1,20 +1,43 @@
 # Based on example Makefile:
 # https://www.cs.swarthmore.edu/~newhall/unixhelp/howto_makefiles.html
+
+# TODO: CXX is a predefined variable. What is its default value on different platforms?
+# TODO: MAKEOPTS += 2 (to parallelize building with 2 commands at once)
 CXX = g++
-SRC_FLAGS = -std=c++0x -I nginx-configparser/
+SRC_FLAGS = -std=c++0x -I nginx-configparser/ -Wall -Wextra
+# TODO: This should be named LDFLAGS for consistency with the Makefile community
 LIB_FLAGS = -lpthread -lboost_system
 PARSER_PATH = ./nginx-configparser/
+GTEST_DIR = nginx-configparser/googletest/googletest
+GTEST_FLAGS = -isystem ${GTEST_DIR}/include
+
+# TODO: Split out dependency handling so that we can avoid rebuilding
+.PHONY: $(TARGET) $(TESTS) all test clean
 
 TARGET = lightning
+TESTS = lightning_server_test
 SRC = $(PARSER_PATH)config_parser.cc lightning_main.cc lightning_server.cc
 
+define \n
```

Comment:
Nice catch! Removed as part of https://github.com/UCLA-CS130/Mr.-Robot-et-al./pull/6/commits/f537957b7c997fd1f9b7258948487955b3c0ee13